types/releases.ts

```diff
@@ -0,0 +1,37 @@
+// TODO(HinataKah0): Remove this, see https://github.com/nodejs/nodejs.org/issues/5303#issuecomment-1518947513
+/* eslint-disable no-unused-vars */
+
+export enum ReleaseTypes {
```

Comment:
Do we need these types? Don't we already have types here? If not that's fine :D

Reply:
Initially I want to move the types to component level but it looks like we can't. After digging, it seems like it's only used in `UpcomingReleasesItem` and `UpcomingReleasesPanel`. However, the data is propagated from page level.

Quick search, I can't find any similar enum.

Reply:
What I mean to say is that we already have download pages? How are we populating these? I guess all statically or something?

But yeah if these types are necessary and used in more than one component then it's okay to leave them here (types/) folder

Reply:
The download page we have is the old one (current nodejs.org).
Should be all statically (GraphQL query once in the top level).
Since we are migrating the new download page, so we need these.